src/Lightning/Node/Api.hs

```diff
@@ -32,6 +32,11 @@ data ApiV1 route = ApiV1
       :> "genInvoice"
       :> ReqBody '[JSON] InvoiceReq
       :> Post '[JSON] InvoiceRep
+  , _listInvoices ::route
+      :- "invoice"
+      :> "listInvoices"
+      :> QueryParam "label" InvoiceLabel
+      :> Get '[JSON] [InvoiceRep]
```

Comment:
It should return `[ListInvoiceRep]`, where `ListInvoiceRep` is a brand new data type, because the data this endpoint returns is very different from what `getnInvoice` returns.